homeassistant/components/tuya/sensor.py

```diff
@@ -790,6 +790,7 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             key=DPCode.TOTAL_FORWARD_ENERGY,
             translation_key="total_energy",
             device_class=SensorDeviceClass.ENERGY,
+            native_unit_of_measurement=UnitOfPower.KILO_WATT,
```

Comment:
Units for these sensors are provided by the API and should not be hardcoded.

../Frenck